2996_reflection/reflection.md

```diff
@@ -4025,9 +4025,13 @@ constexpr int v2 = template [:^^TCls:]<2>::s;
 constexpr typename [:^^TCls:]<3>::type v3 = 3;
   // OK, typename binds to the qualified name
 
-constexpr [:^^TCls:]<3>::type v4 = 4;
-  // error: [:^^TCls:]< is parsed as a splice-expression followed
-  // by a comparison operator
+template [:^^TCls:]<3>::type v4 = 4;
+  // OK, template binds to the splice-scope-specifier
```

Comment:
I thought `typename` was for types, `template` for expressions... this is a type?

Reply:
`template` binds to the splice-scope-specifier, and that is the least gross option that we were able to come up with.